<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Groupbox */

QGroupBox {
    border: 1px solid #DDD;
    border-radius: 9px;
    margin-top: 9px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QGroupBox" name="OptionsGB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Options</string>
      </property>
      <widget class="QPushButton" name="LoadOBJ">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Load OBJ file</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="MeshGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>201</width>
         <height>151</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="loopTitleLabel">
          <property name="text">
           <string>Subdivision</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="stepsLayout">
          <item>
           <widget class="QLabel" name="SubdivLabel">
            <property name="text">
             <string>Steps:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SubdivSteps">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <number>8</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="SubdivisionShadingCheckBox">
          <property name="text">
           <string>Subdivision shading</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QComboBox" name="MeshPresetComboBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OpenCube</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Suzanne</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fertility</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Torus</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tetrahedron</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Octahedron</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Icosahedron</string>
        </property>
       </item>
      </widget>
      <widget class="QGroupBox" name="IsoGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>201</width>
         <height>151</height>
        </rect>
       </property>
       <property name="title">
        <string>Shading</string>
       </property>
       <widget class="QLabel" name="IsoFrequencyLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Frequency:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="IsoSpinBox">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>100</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>91</width>
          <height>74</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="ShadingRadioPhong">
           <property name="text">
            <string>Phong</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ShadingRadioNormal">
           <property name="text">
            <string>Normal</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ShadingRadioIsophote">
           <property name="text">
            <string>Isophote</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="MainView" name="MainDisplay">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainView</class>
   <extends>QOpenGLWidget</extends>
   <header>mainview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
